src/utils/download.ts

```diff
@@ -52,3 +54,16 @@ export async function downloadProject(project: Project, uri?: string) {
     document.body.removeChild(link);
   });
 }
+
+async function addDevKey(queue: Array<{filePrefix: string; file: File}>, uri: string) {
+  const app = (window as any).app;
+  const contractName = `studio-${uri}`;
+  const keyPair = await app.state.keyStore.getKey(contractName);
+  if (keyPair) {
+    const networkId = "devnet"; // TODO: query network id from node
```

Comment:
Looks like `networkId` should be in `keyStore` though judging from tests.

Reply:
Let's do a pass through everything later when we can actually get a real network id. I don't want to polish half baked code since things may change anyways.